types/babel-plugin-macros/index.d.ts

```diff
@@ -1,36 +1,27 @@
-// Type definitions for babel-plugin-macros 2.6
+// Type definitions for babel-plugin-macros 2.8
 // Project: https://github.com/kentcdodds/babel-plugin-macros
 // Definitions by: Billy Kwok <https://github.com/billykwok>
+//                 Jake Runzer <https://github.com/coffee-cup>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
-// TypeScript Version: 2.9
+// TypeScript Version: 3.7
```

Comment:
Is that intended? `@babel/core` works with 2.9:
```ts
// TypeScript Version: 2.9
```
and your change pass linter with that verison as well.

Reply:
oops, not intended. I'll revert.